src/main/java/com/softdevsix/domain/pullrequests/PullRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.softdevsix.domain.pullrequests;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class PullRequest {
```

Comment:
Empty entity, pull request will no longer be used, please check the new workflow. Information will not be received from a pull request now it is received from a static loading project.